generators/app/templates/circle.yml

```diff
@@ -0,0 +1,10 @@
+machine:
+ node:
```

Comment:
Doesn't this need 2 spaces for indentation ? I always have mistaken this point 😅